src/containers/VerifyActivities.jsx

```diff
@@ -84,7 +84,7 @@ class VerifyActivities extends Component {
                           id={activity.id}
                           category={activity.category}
                           date={(activity.date)}
-                          description={activity.activity}
+                          description={activity.description || activity.activity}
```

Comment:
what is the idea here? why or?

Reply:
So some of the activities did not have a description. To handle such cases it was necessary that I included the OR bit.